Error on read property latest

I tried it on windows 8.1 x64.
I cloned, installed the npm packages and launched npm start then I went to http://127.0.0.1:3000/ but it crashed:

``` js
C:\wamp\www\npm-package-store [master +1 ~0 -0 !]> npm start

> npm-package-store@1.0.1 start C:\wamp\www\npm-package-store
> node app

NPM Package Store listening on port 3000.

TypeError: Cannot read property 'latest' of undefined
    at Request._callback (C:\wamp\www\npm-package-store\routes\index.js:25:64)
    at Request.self.callback (C:\wamp\www\npm-package-store\node_modules\request\request.js:372:22)
    at Request.EventEmitter.emit (events.js:98:17)
    at Request.<anonymous> (C:\wamp\www\npm-package-store\node_modules\request\request.js:1310:14)
    at Request.EventEmitter.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (C:\wamp\www\npm-package-store\node_modules\request\request.js:12
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
    at _stream_readable.js:919:16
    at process._tickCallback (node.js:419:13)

npm ERR! npm-package-store@1.0.1 start: `node app`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the npm-package-store@1.0.1 start script.
npm ERR! This is most likely a problem with the npm-package-store package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app
npm ERR! You can get their info via:
npm ERR!     npm owner ls npm-package-store
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\n
npm ERR! cwd C:\wamp\www\npm-package-store
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.9
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\wamp\www\npm-package-store\npm-debug.log
npm ERR! not ok code 0
```

:(

--- a/routes/index.js
+++ b/routes/index.js
@@ -22,10 +22,12 @@ router.get('/', function(req, res) {
           if (registry.author)
             module.author = registry.author.name;
 
-          if (module.installedVersion === registry['dist-tags'].latest) {
+          if (registry['dist-tags'] && module.installedVersion === registry['dist-tags'].latest) {
             upToDate.push(module);
-          } else {
+          } else if (registry['dist-tags']) {
             updates.push(module);
+          } else {
+            console.warn('Skipped package ' + module.name + ' because it does not list a version!');
           }
 
           indexedModules += 1;

